package.json

```diff
@@ -24,8 +25,8 @@
     "ejs": "0.8.4",
     "empty-port": "~0.0.1",
     "es6-promise": "2.0.1",
-    "eslint": "0.21.2",
-    "eslint-plugin-no-unsafe-innerhtml": "0.1.5",
+    "eslint": "1.3.1",
```

Comment:
eslint is [releasing quite often](https://github.com/eslint/eslint/releases), I suggest we take this one and then come up with a plan about how often we want to follow their updates. What do you think?